login/index.html

```diff
@@ -18,22 +18,29 @@
       </a>
 
       <form class="login-form" action="">
-        <label class="login-form__label" for="email">이메일</label>
-        <input id="email" class="login-form__input-email" type="email" placeholder="이메일을 입력해주세요">
-        <label class="login-form__label" for="password">비밀번호
-          <img class="login-form__input-blind" src="/img/btn_visibility_off_24px.png" alt="">
-        </label>
-        <input id="password" class="login-form__input-password" type="password" placeholder="비밀번호를 입력해주세요">
-        <button class="login-form__btn">로그인</button>
+        <div class="login-form__container">
+          <label class="login-form__label" for="email">이메일</label>
+          <input name="email" id="email" class="login-form__input-email" type="email" placeholder="이메일을 입력해주세요">
+        </div>
+        <div class="login-form__container">
+          <div class="login-form__blind-container">
+            <label class="login-form__label" for="password">비밀번호</label>
+            <img class="login-form__input-blind" src="/img/btn_visibility_off_24px.png" alt="">
```

Comment:
img alt들은 제공해주셔야 합니다 
```suggestion
<img class="login-form__input-blind" src="/img/btn_visibility_off_24px.png" alt="비밀번호 표시 버튼" aria-label="비밀번호 표시 전환">

```